### PR TITLE
Clothing Function

### PR DESCRIPTION
**Pull request by Pewwww53**
**Clothing as item**

### CODE REVIEW

File: locales/en.lua [hunk 1]
```diff
@@ -28,6 +28,7 @@ local Translations = {
     info = {
         no_variants = "There don't seem to be any variants for this",
         wrong_ped = "This ped model does not allow for this option",
+        wrong_gender = "This clothe is not for your gender",
```
Comment (TKDubsta1101) [inline locales/en.lua:31]: Spelling

File: server/clothing.lua [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items[item], "remove")
+end)
+
+
+for _, v in ipairs(Config.ClothingItems.clothes) do
+	QBCore.Functions.CreateUseableItem(v, function(source, item)
+		local Player = QBCore.Functions.GetPlayer(source)
+
+		if not Player.Functions.GetItemByName(item.name) then return end
+		print(item.info.model)
```
Comment (TKDubsta1101) [inline server/clothing.lua:21]: Remove debug print

File: server/clothing.lua [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		TriggerClientEvent('qb-radialmenu:WearClothes', source, item, v)
+	end)
+end
+
+for _, v in ipairs(Config.ClothingItems.props) do
+	QBCore.Functions.CreateUseableItem(v, function(source, item)
+		local Player = QBCore.Functions.GetPlayer(source)
+
+		if not Player.Functions.GetItemByName(item.name) then return end
+		print(item.info.model)
```
Comment (TKDubsta1101) [inline server/clothing.lua:31]: Remove Debug Print

File: server/clothing.lua [hunk 4]
```diff
@@ -0,0 +1,34 @@
+RegisterNetEvent('qb-radialmenu:server:giveitem', function(item, amount, slot, info)
+    local Player = QBCore.Functions.GetPlayer(source)
+
+    Player.Functions.AddItem(item, amount, slot, info)
+	TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items[item], "add")
```
Comment (TKDubsta1101) [inline server/clothing.lua:5]: Weird indentation

File: server/clothing.lua [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+end)
+
+RegisterNetEvent('qb-radialmenu:server:removeitem', function(item, amount, slot)
+    local Player = QBCore.Functions.GetPlayer(source)
+
+	Player.Functions.RemoveItem(item, amount, slot)
+	TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items[item], "remove")
```
Comment (TKDubsta1101) [inline server/clothing.lua:12]: Weird indentation

File: server/clothing.lua [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+end)
+
+
+for _, v in ipairs(Config.ClothingItems.clothes) do
+	QBCore.Functions.CreateUseableItem(v, function(source, item)
+		local Player = QBCore.Functions.GetPlayer(source)
+
+		if not Player.Functions.GetItemByName(item.name) then return end
+		print(item.info.model)
+		TriggerClientEvent('qb-radialmenu:WearClothes', source, item, v)
```
Comment (TKDubsta1101) [inline server/clothing.lua:22]: Weird indentation

**Review by TKDubsta1101 — CHANGES_REQUESTED** [review]
Very minor changes but the prints will annoy some people. Possibly add a config for debug prints if you want them in?

**Review by SinisterDevelopment — APPROVED** [review]

**Pull request closed without merging**